googletest/include/gtest/internal/gtest-linked_ptr.h

```diff
@@ -149,12 +149,12 @@ class linked_ptr {
 
   // Take over ownership of a raw pointer.  This should happen as soon as
   // possible after the object is created.
-  explicit linked_ptr(T* ptr = NULL) { capture(ptr); }
+  explicit linked_ptr(T* ptr = NULL) : value_(), link_() { capture(ptr); }
   ~linked_ptr() { depart(); }
 
   // Copy an existing linked_ptr<>, adding ourselves to the list of references.
-  template <typename U> linked_ptr(linked_ptr<U> const& ptr) { copy(&ptr); }
-  linked_ptr(linked_ptr const& ptr) {  // NOLINT
+  template <typename U> linked_ptr(linked_ptr<U> const& ptr) : value_(), link_() { copy(&ptr); }
```

Comment:
I see. The target is to prevent us from having unitialized values from `copy(linked_ptr const&)` behavior changes.